pyramid_orb/api.py

```diff
@@ -159,7 +159,7 @@ def process(self, request):
             output = {}
             for schema in schemas:
                 dbname = schema['dbname']
-                schema['urlRoot'] = request.host_url + request.path.rstrip('/') + '/' + dbname
+                schema['urlRoot'] = request.path.rstrip('/') + '/' + dbname
```

Comment:
@erichulser, well the issue was that when we where serving it from dev, the url route was prepending a https uri scheme name